src/main/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -1182,9 +1181,9 @@ public void extraStrictValidation(final Allele reportedReference, final Allele o
         //checkReferenceTrack();
     }
 
-    public void validateReferenceBases(final Allele reportedReference, final Allele observedReference) {
-        if ( reportedReference != null && !reportedReference.basesMatch(observedReference) ) {
-            throw new TribbleException.InternalCodecException(String.format("the REF allele is incorrect for the record at position %s:%d, fasta says %s vs. VCF says %s", getContig(), getStart(), observedReference.getBaseString(), reportedReference.getBaseString()));
+    public void validateReferenceBases(final Allele expectedReference, final Allele actualReference) {
```

Comment:
@ronlevine I wouldn't change the reported/observed terminology in the whole file.  Just in that one function I was talking about.